packages/protocol/contracts/common/FixidityLib.sol

```diff
@@ -230,17 +226,17 @@ library FixidityLib {
   }
 
   /**
-   * @notice x/y. If the dividend is higher than the max fixed dividend, it
+   * @notice x/y. If the dividend is higher than the max dividend value, it
    * might overflow. You can use multiply(x,reciprocal(y)) instead.
-   * @dev The maximum value that can be safely used as a dividend is defined as
-   * divide(maxFixedDividend,newFixedFraction(1,fixed1())) is around maxUint256().
+   * @dev The maximum value that can be safely used as a dividend (maxNewFixed) is defined as
```

Comment:
Why can you substitute `maxNewFixed` for `maxFixedDividend` everywhere here?

Reply:
they are calculated as the same value (maxUint256()/fixed1()). @mrsmkl suggested simplifying above ^^